plugins/command-line/prism-command-line.css

```
@@ -31,3 +32,12 @@
 .command-line-prompt > span[data-prompt]:before {
 	content: attr(data-prompt);
 }
+
+.command-line-prompt > span[data-continuation-prompt]:before {
+	content: attr(data-continuation-prompt);
+}
+
+.command-line span.token.output {
+  /* Make shell output a bit lighter to distinguish them from shell commands */
```

Please use tabs here as well.
```suggestion
	/* Make shell output a bit lighter to distinguish them from shell commands */
```

---

:facepalm: Sorry.